ckanext/fjelltopp_theme/assets/css/_partials/_fjelltopp.scss

```diff
@@ -0,0 +1,112 @@
+/* global */
+$theme: 'fjelltopp';
+
+/* palette */
+$primary: #002920;
+$secondary: #02A67F;
+$background: #F2F2FC;
+$tag: #7D3667;
+$tag-dark: #7D3667;
+$tag-light: #C770A4;
+$tag-background: #C770A4;
+$dark: #000000;
+$light: #FFFFFF;
+$primary65: #36E07E;
+$primary25: #C4F2DA;
+$primary10: #F2F2FC;
+$neutral-light-grey: #F2F2FC;
+$A11y: #2A2F38;
+
+/* title */
+$default-font-weight: 700;
+$default-font-size: 18px;
+$default-line-height: 20px;
+$default-font-family: 'Inter';
+$default-font-color: #13240F;
+
+/* hide logo left border */
+$hide-logo-left-v-bar: true;
+
+/* promoted */
+$background-color: $primary;
+
+/* insights */
+$show-insights-title: false;
+$upper-container-visible: false;
+$lower-container-visible: true;
+$after-insights-min-height: 400px;
+
+/* show masthead */
+$masthead-visible: true;
+$show-development-banner: true;
+$show-languages: false;
+
+/* recently updated */
+$show-recently-updated-title: false;
+
+/* footer */
+$footer-search-section-padding-top: 64px;
+$footer-search-section-padding-bottom: 0px;
+
+/* badge */
+$badge-font-size: 0.7em;
+
+
+/* for demo purpose only */
+
+.promoted-home {
+    margin: 0;
+    padding: 0;
+    height: 100%;
+    overflow: hidden;
+}
+
+.promoted-home .promoted-container {
+    position: relative;
+    min-height: 500px;
+    z-index: 1;
+}
+
+#background-video {
+    position: absolute;
+    right: 0;
```

Comment:
I think there is something wrong with your responsive design - this is how the home page looks on my screen.

![image](https://github.com/user-attachments/assets/22317207-eb43-44e5-b256-a94a3b8f1cd9)

I think this is the offending line.

Reply:
Oh, and I think it's meant to go behind the title, right? It doesn't for me - tested on latest Chromium and Google Chrome.

Reply:
I have made another scss file that is in charge of loading this background.